modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/KBDropdownPropsTransformer.js

```diff
@@ -44,22 +46,23 @@ const ACTIONS = {
 
 	move(
 		{
-			itemClassNameId,
-			itemId,
-			itemType,
-			moveKBItemActionURL,
-			moveKBItemModalURL,
+			kbObjectClassNameId,
+			kbObjectId,
+			kbObjectTitle,
+			kbObjectType,
+			moveKBObjectActionURL,
+			moveKBObjectModalURL,
 		},
 		portletNamespace
 	) {
 		openSelectionModal({
-			buttonAddLabel: Liferay.Language.get('move'),
+			buttonAddLabel: Liferay.Language.get('save'),
 			height: '50vh',
 			multiple: true,
 			onSelect: ({destinationItem, index}) => {
 				if (
-					itemType === ITEM_TYPES.folder &&
-					destinationItem.type === ITEM_TYPES.article
+					kbObjectType === ITEM_TYPES.KBFolder &&
+					destinationItem.type === ITEM_TYPES.KBArticle
```

Comment:
Isn't the `ITEM_TYPES` thing a bit redundant? As the name of the constant is the same as the value, this only adds typing effort (IMO).